src/corelibs/U2View/src/ov_sequence/PanView.cpp

```diff
@@ -308,7 +308,7 @@ void PanView::sl_onAnnotationSettingsChanged(const QStringList& changedSettings)
     AnnotationSettingsRegistry* asr = AppContext::getAnnotationsSettingsRegistry();
     foreach (const QString& name, changedSettings) {
         AnnotationSettings* as = asr->getAnnotationSettings(name);
-        bool hasRow = rowsManager->contains(name);
+        bool hasRow = rowsManager->containsAnnotationWithName(name);
```

Comment:
nit: it is better to call this method: hasRowWithName()  because rows are named differently from annotations (enzymes)